### PR TITLE
chore(deps): bulk updated all the things

### PR DESCRIPTION
**Pull request by tkrugg**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -126,23 +128,25 @@
...
     "readline-sync": "1.4.9",
     "replace-in-file": "3.4.3",
-    "rxjs": "5.5.12",
+    "rxjs": "6.4.0",
     "semver": "5.6.0",
     "shelljs": "0.8.3",
     "style-loader": "0.23.1",
     "ts-loader": "5.3.3",
     "tsickle": "0.34.0",
+    "tslib": "1.9.0",
```
Comment (tkrugg) [inline package.json]: this is a new peer dependency required by ng-packagr@4.7.0
https://cdn.jsdelivr.net/npm/ng-packagr/package.json

File: src/current-refinements/__tests__/__snapshots__/current-refinements.spec.ts.snap [hunk 2]
```diff
@@ -62,7 +62,7 @@ exports[`CurrentRefinedValues renders markup with state 1`] = `
             <span
               class="ais-CurrentRefinements-label"
             >
-              Hierarchicalcategories.Lvl0:
+              Hierarchicalcategories.lvl0:
```
Comment (francoischalifour) [inline src/current-refinements/__tests__/__snapshots__/current-refinements.spec.ts.snap]: Do you know why this has changed?

Comment (tkrugg) [reply src/current-refinements/__tests__/__snapshots__/current-refinements.spec.ts.snap]: yes, there has been some bug fixes in the titlecase pipe in Angular 6: https://github.com/angular/angular/blob/master/CHANGELOG.md#bug-fixes-39

I think the commit where this happened is here https://github.com/angular/angular/commit/7966744

No impact on us.

File: package.json [hunk 3]
```diff
@@ -126,23 +128,25 @@
...
     "tslint": "5.12.1",
     "tslint-config-airbnb": "5.11.1",
     "tslint-config-prettier": "1.17.0",
     "tslint-plugin-prettier": "2.0.1",
     "typescript": "3.2.4",
+    "webpack": "4.3.0",
```
Comment (Haroenv) [inline package.json]: what's the reason we are adding webpack here. Could we instead leverage ["resolutions" (the Yarn feature)](https://yarnpkg.com/en/docs/selective-version-resolutions#search)

Comment (tkrugg) [reply package.json]: it's a peerDependency of css-loader, raw-loader, ... which I don't think we need actually.
let's me check if I get rid of them.

Comment (tkrugg) [reply package.json]: removed it along with all the dependencies that turned out to be unsed https://github.com/algolia/angular-instantsearch/pull/470/commits/c5b6b3ce2ffb2e3bf1f22d46b767ce5723e6193d

**Pull request merged**